modules/geolocation/unittests.py

```diff
@@ -203,6 +203,134 @@ def test_point_set_2(self):
         # Test
         np.testing.assert_almost_equal(actual, expected)
 
+class TestGetBestLocation(unittest.TestCase):
+    """
+    Tests Geolocation.outputanalysis(inputLocationTupleList)
+    """
+    def test_all_ones(self):
+
+        # Setup
+        outputAnalysis = geolocation.Geolocation()
+
+        listToAnalyze = np.array([[[1, 1], [1], [1]]])
+
+        expectedCoordPair = np.array([[1,1]])
+        expectedError = np.array([[1]])
+
+        # Run
+        actual = outputAnalysis.get_best_location(listToAnalyze)
+
+        np.testing.assert_almost_equal(actual[0], expectedCoordPair)
+        np.testing.assert_almost_equal(actual[1], expectedError)
+
+    def test_all_zeros(self):
+
+        # Setup
+        outputAnalysis = geolocation.Geolocation()
+
+        listToAnalyze = np.array([[[0,0],[0],[0]]])
+
+        expectedCoordPair = np.array([[0,0]])
+        expectedError = np.array([[0]])
+
+        # Run
+        actual = outputAnalysis.get_best_location(listToAnalyze)
+
+        np.testing.assert_equal(actual[0], expectedCoordPair)
+        np.testing.assert_equal(actual[1], expectedError)
+
+    def test_normal_input(self):
+
+        # Setup
+        outputAnalysis = geolocation.Geolocation()
+
+        listToAnalyze = np.array([
+        [[10, 20], [2.3], [0.5]],
+        [[11, 19], [2.2], [0.7]],
+        [[12, 21], [2.1], [0.9]]])
+
+        expectedCoordPair = (np.array([[11],[20]]))
+        expectedError = (np.array([2.2]))
+
+        # Run
+        actual = outputAnalysis.get_best_location(listToAnalyze)
+
+        np.testing.assert_almost_equal(actual[0], expectedCoordPair)
+        np.testing.assert_almost_equal(actual[1], expectedError)
+
+    def test_input_with_one_outlier(self):
```

Comment:
Rather than "input", you could use "point" or "coordinate" (e.g. `test_point_with_one_outlier()` or `test_coordinate_with_one_outlier()`), because at the moment you're not testing the errors, which I saw was further below.